make/stanc

```diff
@@ -24,24 +24,40 @@ else
 endif
 
 ifeq ($(CMDSTAN_SUBMODULES),1)
-ifeq ($(STANC3),)
-bin/stanc$(EXE) :
+# bin/stanc build rules
+ifneq (,$(STANC2))
```

Comment:
I think it would be best if both conditions were written in the same order:

`ifneq (,$(STANC2))`
`ifneq ($(STANC3),)`